### ISSUE TITLE
Cannot access /ext/vm/avm endpoint

### ISSUE THREAD

**Issue by etkinpinar** [ava-labs/avalanchego#817]
**Describe the bug**
Cannot send request to ext/vm/avm endpoint where it returns 404 Page not found.

**To Reproduce**
```bash
curl -X POST --data '{
    "jsonrpc": "2.0",
    "id"     : 1,
    "method" : "avm.buildGenesis",
    "params" : {
        "genesisData": {
            "BZAsset": {
                "name": "MyAsset",
                "symbol":"BZA",
                "initialState": {
                    "variableCap" : [
                        {
                            "minters": [
                                "local1mvv8jnk87ak9hm2uzhugnfxn7rphpfgkqmqsgx",
                                "local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"
                            ],
                            "threshold":1
                        }
                    ]
                }
            }
        }
        }
}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/vm/avm
```
**Returns**
`404 page not found`

**Operating System**
Ubuntu 18.04

**Additional context**
I was trying to create a blockchain on my local network and I was following the docs but I don't get what's happening here. It responds like there is no endpoint such as ext/vm/avm. If the endpoint I am trying to reach is invalid, what is the accurate endpoint for avm.buildGenesis? Thank you for response in advance.

By submitting this issue I agree to the [Terms and Conditions](https://files.avalabs.org/legal/bug_hunt_q2_2020_terms.pdf) of the [Developer Accelerator Program](https://avalabs.org/dap).

**Comment by StephenButtolph** [comment]
Thanks for finding this. It seems that `/ext/vm/avm` isn't currently accessible. This will be fixed in `v1.3.2`. Sorry for the inconvenience.

**Comment by etkinpinar** [comment]
I've found out something that might cause this bug but I'm not able to fix this on my own. Therefore, I thought it might be useful for you devs if I share it here. What I've found out about this issue is that type assertion on staticVM (in `vms/manager.go` file and `addStaticAPIEndpoints` function) fails for some reason and that causes the issue. And that failure does not occur for timestampvm and platformvm.

**Code Snippet**
``` go
func (m *manager) addStaticAPIEndpoints(vmID ids.ID) {
	vmFactory, err := m.GetVMFactory(vmID)
	m.log.AssertNoError(err)
	m.log.Debug("adding static API for VM with ID %s", vmID)
	vm, err := vmFactory.New(nil)
	if err != nil {
		return
	}

	staticVM, ok := vm.(common.StaticVM) // << This is the part I'm trying to refer 

	if !ok {
		staticVM, ok := vm.(common.VM)
		if ok {
			if err := staticVM.Shutdown(); err != nil {
				m.log.Error("shutting down static API endpoints errored with: %s", err)
			}
		}
		return
	}

	// all static endpoints go to the vm endpoint, defaulting to the vm id
	defaultEndpoint := "vm/" + vmID.String()
	// use a single lock for this entire vm
	lock := new(sync.RWMutex)
	// register the static endpoints
	for extension, service := range staticVM.CreateStaticHandlers() {
		m.log.Verbo("adding static API endpoint: %s", defaultEndpoint+extension)
		if err := m.apiServer.AddRoute(service, lock, defaultEndpoint, extension, m.log); err != nil {
			m.log.Warn("failed to add static API endpoint %s: %v", fmt.Sprintf("%s%s", defaultEndpoint, extension), err)
		}
	}
}
```

**Comment by elmurci** [comment]
Is there any workaround to build a genesis block in the meantime?

**Comment by aaronbuchwald** [comment]
Closing this since it has been fixed as of v1.3.2.
